### PR TITLE
Add tests for roles playwright

### PR DESCRIPTION
**Pull request by sweta1308**
### Describe your changes:
Add tests for Roles in playwright.

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  expect(modalText).toBe(
+    `Are you sure you want to remove the ${policyName} from ${roleName}?`
+  );
+
+  await page.locator('[type="button"]:has-text("Confirm")').click();
+};
+
+const navigateToRolesPage = async (page: Page) =>
+  await page.goto('/settings/access/roles');
+
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts:46]: lets move to utils

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  expect(modalText).toBe(
+    `Are you sure you want to remove the ${policyName} from ${roleName}?`
+  );
+
+  await page.locator('[type="button"]:has-text("Confirm")').click();
+};
+
+const navigateToRolesPage = async (page: Page) =>
+  await page.goto('/settings/access/roles');
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts:45]: @sweta1308 , can we use `settingClick` util here?

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts (hunk 1 above)
Comment (sweta1308) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts]: Updated

File: openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts (hunk 2 above)
Comment (sweta1308) [reply openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Roles.spec.ts]: Updated

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Review by ShaileshParmar11 — DISMISSED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**